styles/stylesheet.css

```diff
@@ -0,0 +1,234 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  /*font-family*/
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-image: url("../images/header-image.jpg");
+  padding: 7em;
+  color: #55C632;
+}
+
+.float-right {
+  float: right;
+  width: 5%;
+}
+
+header.logo {
+  clear: both;
+}
+
+header button {
+  background-color: #55C632;
+  color: #FFFFFF;
+  font-weight: bolder;
+}
+
+nav {
+  background-color: #DCDCDC;
+  padding: 0.5em;
+}
+
+nav ul li {
+  display: inline;
+  margin: 0.5em;
+  color: #154360;
+}
+
+a {
+  text-decoration: none;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.home-intro, .blog-post {
+  margin: 2em 6.5em 1em 6.5em;
+  display: inline-block;
+}
+
+.home-intro img, .blog-post img {
+  float: left;
+  width: 30%;
+}
+
+.home-intro p, .home-intro h1, .blog-post p, .blog-post h2{
+  margin: 1em 2em;
+  display: table;
```

Comment:
Don't use display table for this kind of thing. We only want to use display table when we are displaying tables of data